aten/src/ATen/native/native_functions.yaml

```diff
@@ -10965,10 +10917,6 @@
 
 - func: upsample_nearest3d.vec(Tensor input, int[]? output_size, float[]? scale_factors) -> Tensor
   python_module: nn
-  dispatch:
-    CPU: upsample_nearest3d_cpu
-    CUDA: upsample_nearest3d_cuda
-    QuantizedCPU: upsample_nearest3d_quantized_cpu
   autogen: upsample_nearest3d.vec_out
 
 - func: _upsample_nearest_exact3d.vec(Tensor input, int[]? output_size, float[]? scale_factors) -> Tensor
```

Comment:
Also, what is it with these?

Reply:
`interpolate` will call different versions of all of these guys depending on what params you pass it. E.g,

https://github.com/pytorch/pytorch/blob/30fb2c4abaaaa966999eab11674f25b18460e609/torch/csrc/api/include/torch/nn/functional/upsampling.h#L145

I will clean up these next, once I can check that there's no unexpected problem from the changes so far.